content/en/docs/tasks/security/authorization/authz-dry-run/index.md

```diff
@@ -22,7 +22,7 @@ Before you begin this task, do the following:
 
 * Read the [Istio authorization concepts](/docs/concepts/security/#authorization).
 
-* Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio.
```

Comment:
may be this was intentional. As istioctl is the recommended way of installation? @craigbox @dhawton

Reply:
istioctl for quick demos, helm for production.. so one isn't preferred over another.